pyproject.toml

```diff
@@ -13,6 +13,7 @@ regex = "^2020.7.14"
 retry = "^0.9.2"
 paramiko = "^2.7.1"
 singleton-decorator = "^1.0.0"
+spurplus = "^2.3.3"
```

Comment:
Hey Spur+ looks pretty cool!

Reply:
Yes, it works. It can be better, if it supports same call to Paths for local session.